### ISSUE TITLE
Export name in readme does not work together with HTML <menu> tag

### ISSUE THREAD

**Issue by johanrd** [hokulea/aria-voyager#170]
From the readme:

```
import { menu } from 'ember-aria-voyager';
```

If a menu tag is used in the same component:
```gts
<menu {{menu}}>
```

the following runtime error pops up:

```
Error: Attempted to load a component, but there wasn't a component manager associated with the definition. The definition was: MenuModifier
```

Changing to a different import name fixes it.
```ts
import { menu as menuModifier } from 'ember-aria-voyager';
//or
import menuModifier from 'ember-aria-voyager/modifiers/menu';
```

Perhaps the readme should include one of these instead, as they are more generally usable? Default naming is hard, though. `menuModifier`, `menuAria`, `menuAriaModifier`, `ariaVoyager`…

**Comment by gossi** [comment]
oh, this is really good.

I was using `<menu>` in combination with the now deprecated `<command>` element - so I forget about the former - which is actually the more accurate/semantic element for the use-case at hand.

Thanks so much.

For the issue itself, I will forward it to glimmer, too. I would even think that this should be fine code, tbh. You brought up an interesting case here.

Also think about different names, too.

Thanks, interesting case here 👍

**Comment by gossi** [comment]
I prefixed ember modifiers with `aria` in #270

Released in `0.1.0`
